contracts/facets/DFMoveFacet.sol

```diff
@@ -109,6 +113,13 @@ contract DFMoveFacet is WithStorage {
         }
 
         _executeMove(args);
+        
+        if(arenaStorage().startTime == 0) {
+           arenaStorage().startTime = block.timestamp; 
+           emit GameStarted(msg.sender);
+        }
+        ArenaPlayerInfo storage arenaPlayer = arenaStorage().arenaPlayerInfo[msg.sender];
```

Comment:
can you do this in one line? `arenaStorage().arenaPlayerInfo[msg.sender].moves++` ?

Reply:
not sure if it matters in runtime, im sure the compiler handles that

Reply:
ok